### DISCUSSION TITLE
When a large file appears in search that might or not be a n64 zip the search  freezes when you cancel it it will start canceling but it will stay in that position please fix this if it is a bug

### DISCUSSION THREAD

**Discussion by Danzie456** [mupen64plus-ae/mupen64plus-ae#991] [Q&A]
Will you be able to fix it?

**Comment by fzurita** [comment]
Are you in the latest app version? It shouldn't matter how big the zip file is, it will only scan up to 10 files before deciding that there are no ROMs in that zip file.

This did not use to be the case in the past, so that's why I need to make sure you are in the latest app version.

How long after you cancel does it take to actually cancel?
